src/components/MapView/Layers/ImpactLayer/index.tsx

```diff
@@ -23,17 +24,17 @@ import {
   mapSelector,
 } from '../../../../context/mapStateSlice/selectors';
 import { getFeatureInfoPropsData, getRoundedData } from '../../utils';
+import { i18nTranslator, safeTranslate } from '../../../../i18n';
 
 const linePaint: LinePaint = {
   'line-color': 'grey',
   'line-width': 1,
   'line-opacity': 0.3,
 };
 
-function getHazardData(evt: any, operation: string) {
+function getHazardData(t: i18nTranslator, evt: any, operation: string) {
```

Comment:
chance we can fix the any type while we're here?

Reply:
I can, but looking into it, i think it might result from not imposing typing in other props updating functions like `getFeatureInfoPropsData`, so I prefer to do this in a different refactoring work item